### ISSUE TITLE
SEO for Giveth Visibility on Search Engines

### ISSUE THREAD

**Issue by ahmadabugosh** [Giveth/giveth-planning#1177]
We need to create landing pages for SEO purposes to drive traffic to Giveth based on key searches.

[Using this list](https://docs.google.com/spreadsheets/d/10e9ltdCl16rXgZ82gxAEZ-ROlhQiSnnIItUKfSaGUgc/edit#gid=325379001), we can determine which keywords to prioritize and work on getting these pages implemented.

Once the comms work is done to set up these pages, a new issue needs to be created in the dApps board to get the design work started.
